src/components/table/index.tsx

```diff
@@ -201,78 +201,76 @@ export default function EnhancedTable<T>({
   return (
     <>
       {showTopBar && <EnhancedTableToolbar numSelected={selectedRows ? selectedRows.length : 0} topBarButtons={topBarButtons} />}
-      <TableContainer id={id}>
-        <DndContext sensors={sensors} onDragEnd={handleDragEnd}>
-          <Table stickyHeader={true} aria-labelledby='tableTitle' size='medium' aria-label='enhanced table' className={classes.table}>
-            <TableHeader
-              order={order}
-              orderBy={orderBy}
-              onRequestSort={handleRequestSort}
-              columns={columns}
-              onReorderEnd={onReorderEnd}
-              numSelected={showCheckbox ? selectedRows?.length : undefined}
-              onSelectAllClick={showCheckbox ? handleSelectAllClick : undefined}
-              rowCount={showCheckbox ? rows?.length : undefined}
-            />
-            <TableBody>
-              {rows.length < 1 && emptyTableMessage && (
-                <TableRow>
-                  <TableCell colSpan={columns.length + 1} align='center'>
-                    <p>{emptyTableMessage}</p>
-                  </TableCell>
-                </TableRow>
-              )}
-              {rows &&
-                stableSort(rows, getComparator(order, orderBy, sortComparator))
-                  .slice(itemsToSkip, itemsToSkip + maxItemsPerPage)
-                  .map((row, index) => {
-                    const onClick = onSelect && !controlledOnSelect ? () => onSelect(row as T) : undefined;
-                    const isItemSelected = isSelected(row as T);
+      <DndContext sensors={sensors} onDragEnd={handleDragEnd}>
+        <Table stickyHeader={true} aria-labelledby='tableTitle' size='medium' aria-label='enhanced table' className={classes.table}>
```

Comment:
Are all our tables going to have sticky header?

Reply:
This would turn it on for all tables by default, yes. I can change it to a prop that we pass in for only those tables we want this turned on for.

Reply:
Maybe this is ok. Not sure what the requirement is. We could check with @minqingzhou

Reply:
I went ahead and added the prop. it will default to non-sticky

Reply:
I can change that 👍